tests/ut_team.py

```diff
@@ -0,0 +1,35 @@
+import team
+import unittest
+
+class TestTeamLogin(unittest.TestCase):
+    def setUp(self):
+        self.team1 = team.Team("Name1")
+        self.team2 = team.Team("Name2")
+    def test_setTeamName(self):
+        self.assertEquals("Team name changed to n1", self.team1.setTeamName("n1"), "Error: changing team name failed")
```

Comment:
Should be written like this:

def test_setTeamName(self):
    self.team1.set_team_name("n1")
    self.assertEquals(self.team1, "n1", "Error Message")

Same for below tests. Error messages are printed to console, not returned by these functions.